lib/src/Pos/Communication/Limits.hs

```diff
@@ -317,6 +319,15 @@ mlBlock bvd = mlEither (mlGenesisBlock bvd) (mlMainBlock bvd)
 mlMsgBlock :: BlockVersionData -> Limit MsgBlock
 mlMsgBlock = fmap MsgBlock . mlBlock
 
+mlMsgStream :: Limit MsgStream
+mlMsgStream = mlMsgStreamStart
+
+mlMsgStreamStart :: Limit MsgStream
+mlMsgStreamStart = 0x7ffff -- XXX A 512k limit, if these numbers are bytes.
+
+mlMsgStreamBlock :: BlockVersionData -> Limit MsgStreamBlock
+mlMsgStreamBlock = fmap MsgStreamBlock . mlBlock -- XXX MsgStreamNoBlock has an arbitrary text field...
```

Comment:
Have I correctly inferred that "XXX" is your shorthand for "TODO"?

Reply:
That would be short hand for "Beware".

Reply:
Oh, it's apparently well known! (Just not by me) https://en.wikipedia.org/wiki/XXX

> XXX, a comment to warn other programmers of problematic or misleading code

Reply:
I didn't know that `XXX` is used in this context either :)